cfgov/unprocessed/css/misc.less

```diff
@@ -670,11 +670,11 @@
     - cfgov-misc
 */
 .u-move-transition {
-    transition: transform 0.25s ease-out;
+    transition: transform 0.25s linear;
 }
 
 .u-alpha-transition {
-    transition: opacity 0.25s ease-out;
+    transition: opacity 0.25s linear;
```

Comment:
The rest of the PR may be helpful, but please revert the changes to easing methods (here and L673). We chose `ease-out` intentionally.